lib/exact/index.ts

```diff
@@ -87,6 +87,10 @@ type ArrayExact<TValue extends readonly any[], TShape extends readonly any[]> =
 // `isBCorBC3(bcOrBc3)` fails because `bcOrBc3` is inferred as `BC`???
 type PrimitiveExact<TValue, TShape> = [TValue] extends [TShape] ? ([TShape] extends [TValue] ? TValue : never) : never;
 
+
+/**
+ * @deprecated `Exact` will be removed in v11.0. Please use `IsExact`
+ */
```

Comment:
Could you also remove README for Exact and rename its occurrences to IsExact in top README?

Reply:
Although last item is optional, as you're changing `test/index.ts`, I recommended to move testing to another existing file in https://github.com/ts-essentials/ts-essentials/pull/457#discussion_r2463114483. We can close this conversation